### PR TITLE
Reviewed and edited styleguide sections

### PR DESCRIPTION
**Pull request by dispera**
Scope
=====

Continue to review and provide feedback for the styleguide as part of my onboarding process: chevah/onboarding#23,

This review specifically covers Bugs/Leave/Library/Python/Release/Work sections - and a really quick add to Review section.

Changes
=======

I corrected gramatical errors, and also changed the way some parts were explained or spelled.

Reviewers
===============================

reviewers: @adiroiban

### CODE REVIEW

**Comment by dispera** [comment]
@adiroiban Please remember this PR from Friday when you can, seems it got lost in the shuffle, it has many fixes :)

**Comment by dispera** [comment]
needs-review

**Comment by adiroiban** [comment]
@hcs0 please review this.

@dispera if Hannah is happy with it, please merge. Thanks!

File: content/work.rst [hunk 1]
```diff
@@ -194,14 +193,14 @@ Well documented, small tasks based work
 It feels flattering to be essential to the project.
 But that usually also means that project won’t make rapid progress without you.
 
-This is not a good thing. Both for you and the project.
-What happens if want to take a (spontaneous) vacation or get sick?
-Or some work gets delayed and overlaps with your great vacation, scheduled many months before.
+This is not a good thing. Both for you and for the project.
+What happens if want to take a (spontaneous) vacation or if you get sick?
```
Comment (hcs0) [inline content/work.rst]: Add _you_ in this line: _What happens if you want to take a (spontaneous) vacation or if you get sick?_

File: content/defects-tech-debt.rst [hunk 2]
```diff
@@ -17,28 +17,28 @@ In the same time, we aim to have the perfect development process, with the
...
-Defects are defined as behavior of our product which does not follow our
+Defects are defined as unintended behavior of our product which does not follow our
 specification or the way we want it to work.
 
-Mis-behaviors in our build system, testing system or infrastructure are also
+Misbehaviors in our build system, testing system or infrastructure are also
 defects.
 
-We split the defect into 3 categories:
+Defect can fall into 3 categories:
```
Comment (hcs0) [inline content/defects-tech-debt.rst]: Add _can_ (or even _may_):  _Defects can fall into 3 categories:_

**Comment by dispera** [comment]
@hcs0 Thanks! Corrected those parts now :)

**Pull request merged**